music21/pitch.py

```diff
@@ -5409,7 +5408,7 @@ def testAccidentalsCautionary(self):
         bm.makeNotation(inPlace=True, cautionaryNotImmediateRepeat=False)
         notes = bm[note.Note]
         self.assertEqual(notes[0].pitch.accidental.name, 'natural')     # Fn
-        self.assertEqual(notes[0].pitch.accidental.displayStatus, False)
+        self.assertEqual(notes[0].pitch.accidental.displayStatus, True)
```

Comment:
This change in 8e46d3f reverts to the original state of this test before the PR. I didn't notice the test was inserting a key of 1 sharp, which should have alerted me that there was a flaw in the implementation. So in sum, no changes to existing tests.